app/src/main/java/com/duckduckgo/app/bookmarks/ui/SavedSiteDialogFragment.kt

```diff
@@ -39,7 +38,7 @@ import com.duckduckgo.app.browser.databinding.DialogFragmentSavedSiteBinding
 import com.duckduckgo.app.global.view.TextChangedWatcher
 import com.duckduckgo.mobile.android.R as CommonR
 import com.duckduckgo.mobile.android.ui.view.showKeyboard
-import kotlinx.android.synthetic.main.include_find_in_page.*
+import com.duckduckgo.mobile.android.ui.view.text.DaxTextInput
 
 abstract class SavedSiteDialogFragment : DialogFragment() {
```

Comment:
Can you move the style in https://github.com/duckduckgo/Android/blob/a576fac4a4d287dec18e2cacb6c87961a280f392/app/src/main/java/com/duckduckgo/app/bookmarks/ui/SavedSiteDialogFragment.kt#L81 to widgets.xml and `Widgets.DuckDuckGo.DialogFullScreen`